src/components/dashboard/news-widget.vue

```diff
@@ -23,13 +23,13 @@ export default {
 </script>
 
 <template lang="pug">
-widget.news-widget(title="News")
-  div( style="max-height: 500px; overflow: auto;" ref="scrollTargetRef")
+widget.news-widget(title="Latest News")
+  div( style="max-height: 400px; overflow: auto;" ref="scrollTargetRef")
     q-infinite-scroll(@load="onLoad" :offset="250" :scroll-target="$refs.scrollTargetRef")
       template(v-slot:loading)
         div(class="row justify-center q-my-md")
           q-spinner-dots(color="primary" size="40px")
-      news-item.q-px-sm(v-for="newsItem in news" :key="news.title" :news="newsItem")
+      news-item.q-px-sm(v-for="newsItem in news" :key="news.title" :title="newsItem.title" :date="newsItem.date" :description="newsItem.description" :author="newsItem.author" :tags="newsItem.tags")
```

Comment:
This `key` value looks wrong - should be `newsItem.title` ??

You can have vue automatically bind the properties of an object to the props so you don't need to specify them all by doing:
`news-item.q-px-sm(v-for="newsItem in news" :key="newsItem.title" v-bind="newsItem")`